### PR TITLE
Added 'design_name' and 'orfs_platform' parameters to the ORFS steps

### PR DESCRIPTION
**Pull request by dolendravikas08**
- Added following items to the parameters section in .yml files of the ORFS steps
  - design_name: undefined
  - clock_period: 1.0
  - adk: undefined

- Appended make command with `DESIGN_CONFIG=./designs/$adk/$design_name/config.mk`

- Modified the `construct-open.py` with the appropriate 'adk' and 'design_names' 

- Added 'ibex' (a small 32 bit RISC-V CPU core with a two stage pipeline) to the example designs

### CODE REVIEW

File: designs/GcdUnit/construct-open.py [hunk 1]
```diff
@@ -19,12 +19,13 @@ def construct():
...
+  # adk_name = 'freepdk-45nm'
+  adk_name = 'nangate45'
   adk_view = 'view-tiny'
 
   parameters = {
     'construct_path' : __file__,
-    'design_name'    : 'GcdUnit',
+    'design_name'    : 'gcd',
     'clock_period'   : 2.0,
     'adk'            : adk_name,
```
Comment (ctorng) [inline designs/GcdUnit/construct-open.py:30]: Add `orfs_platform` here

File: designs/GcdUnit/construct-open.py [hunk 2]
```diff
@@ -19,12 +19,13 @@ def construct():
   # Parameters
   #-----------------------------------------------------------------------
 
-  adk_name = 'freepdk-45nm'
+  # adk_name = 'freepdk-45nm'
+  adk_name = 'nangate45'
```
Comment (ctorng) [inline designs/GcdUnit/construct-open.py:23]: Revert this

File: steps/orfs-docker-setup/configure.yml [hunk 3]
```diff
@@ -39,6 +40,9 @@ commands:
 #-------------------------------------------------------------------------
 
 parameters:
+  design_name: undefined
+  clock_period: 1.0
+  adk: undefined
```
Comment (ctorng) [inline steps/orfs-docker-setup/configure.yml:45]: orfs-docker-setup does not use these, so remove these...

File: steps/orfs-yosys-synthesis/configure.yml [hunk 4]
```diff
@@ -37,9 +37,11 @@ commands:
   - for x in inputs/orfs-design/*; do
       docker cp -aL $x $CONTAINER:/OpenROAD-flow-scripts/flow/designs/;
     done
+  # Check design name and adk
+  - echo -e "\n- design_name -- $design_name adk_name -- $adk"
   # Execute
   - echo -e "\n- Executing flow commands inside orfs docker container -- $CONTAINER\n"
-  - docker exec $CONTAINER bash -c "cd flow && make synth"
+  - docker exec $CONTAINER bash -c "cd flow && make synth DESIGN_CONFIG=./designs/$adk/$design_name/config.mk"
```
Comment (ctorng) [inline steps/orfs-yosys-synthesis/configure.yml:44]: Change to $orfs_platform

**Review by ctorng — CHANGES_REQUESTED** [review]

**Comment by dolendravikas08** [comment]
Added "orfs_platform" as a parameter instead of using "adk".

**Comment by ctorng** [comment]
Looks great! Go ahead and click rebase and merge!

**Pull request merged**